0.5/articles/unit-testing-elements.md

```diff
@@ -205,7 +205,7 @@ In this tutorial, we're going to add an existing Polymer element, [`<core-select
 
 First, get `<seed-element>` setup in a new `development` directory (if you haven't already in the Quick Start):
 
-    # Create a new directory to work in
+    # Create a new directory to work in (?)
```

Comment:
Is this really necessary? Almost all devs will already have such a directory (probably named something else), and the 'clone' makes a self-contained directory anyway.